src/platform/packages/shared/kbn-unified-tabs/src/hooks/use_responsive_tabs.tsx

```diff
@@ -71,9 +71,9 @@ export const useResponsiveTabs = ({
   useEvent('scroll', onScrollThrottled, tabsContainerElement);
 
   useEffect(() => {
-    onScrollThrottled();
-    // `isScrollable` added here to trigger in cases when the container width changes
-  }, [tabsContainerElement, onScrollThrottled, tabsSizeConfig.isScrollable]);
+    onScroll();
+    // `dimensions.width` added here to trigger in cases when the container width changes
+  }, [onScroll, dimensions.width]);
```

Comment:
Another existing issue. Previously the overflow shadow would sometimes display when it shouldn't due to a combo of throttling and updating based on `tabsSizeConfig.isScrollable` instead of the actual container width. Now it skips throttling for these calls and updates whenever the width changes.